probes/hasDangerousWorkflowScriptInjection/patch/testdata/twoInjectionsSameJob_fixed.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+on:
+  discussion:
+    types: [created]
+
+env:
+  DISCUSSION_BODY: ${{ github.event.discussion.body }}
+  DISCUSSION_TITLE: ${{ github.event.discussion.title }}
```

Comment:
Good one! I don't think Scorecard actually detects this... please suggest it in an issue there!

Reply:
https://github.com/ossf/scorecard/issues/3915